gui/budget_panel.gui

```diff
@@ -1376,7 +1376,6 @@ types budget
 								fontsize = 45
 								autoresize = yes
 								margin = { 10 10 }
-								align = center
```

Comment:
Why are you removing the `align` segments? Are they invalidated by some other bits in there?

Reply:
In this case there is another align right above it. In other cases the error log reports that they are not used.

Reply:
Have you checked it out ingame? Does it look fine without the aligns?

Reply:
Just to reiterate this one had two aligns = center

Reply:
Yea, I'm more so interested in the ones without any aligns at all now :p

Reply:
I only briefly checked the vienna congress